apis/r/R/utils-arrow.R

```diff
@@ -88,6 +89,10 @@ tiledb_type_from_arrow_type <- function(x) {
     dictionary = "INT32", 			# for a dictionary the 'values' are ints, levels are character
```

Comment:
This seems to miss the PR I just made / which is pending ie #1835.  Maybe get that one out of the way first and / or rebase against it ?

Reply:
@eddelbuettel they're separate PRs, each fixing what it fixes -- I am OK merging them both separately